security/s2a/internal/crypter/counter_test.go

```diff
@@ -0,0 +1,109 @@
+package crypter
+
+import (
+	"math"
+	"testing"
+)
+
+type counterTest struct {
+	desc                     string
+	counter, expectedCounter uint64
+	overflow                 bool
+}
+
+func TestCounterInc(t *testing.T) {
+	for _, test := range []counterTest{
```

Comment:
The two tests don't seem to use the exact same struct. I recommend removing the struct on line 8 and integrate it directly in the tests.

```
for _, tc := range []struct{
   ...
} {

}{

}
```

Reply:
Done